### DISCUSSION TITLE
When I use batch images, it will prompt an error "OSError: cannot write mode I;16 as JPEG"

### DISCUSSION THREAD

**Discussion by dioking** [thygate/stable-diffusion-webui-depthmap-script#136] [Q&A]
<img width="923" alt="b282e1be6e9d733bd8dc6bdee58b117" src="https://user-images.githubusercontent.com/32161007/223588201-247ea87a-bd19-4f89-bb13-5b33b378c428.png">
When I use batch images, it will prompt an error "OSError: cannot write mode I;16 as JPEG"

**Comment by dioking** [comment]
In addition, the first frame of the depth map will be generated while the batch processing interrupt prompts an error

**Comment by thygate** [comment]
>"OSError: cannot write mode I;16 as JPEG" 

It's trying to save a 16bit grayscale image as jpeg, which is not supported in that format, please use png.
